contributors/devel/sig-testing/integration-tests.md

```diff
@@ -71,7 +71,7 @@ to run a specific integration test case:
 
 ```sh
 # Run integration test TestPodUpdateActiveDeadlineSeconds with the verbose flag set.
-make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"
+make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds\$\$\$\$"
```

Comment:
I can see why `"-run ^TestPodUpdateActiveDeadlineSeconds$"` is wrong.

But why not simply use this?

```suggestion
make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS='-run ^TestPodUpdateActiveDeadlineSeconds$'
```

If that doesn't work, then we have a problem with string handling in the k/k test scripts which we should fix.


```

Reply:
`KUBE_TEST_ARGS='-run ^TestPodUpdateActiveDeadlineSeconds$'` not work, we need  `KUBE_TEST_ARGS='-run ^TestPodUpdateActiveDeadlineSeconds$$$$'`

Reply:
That's a deficiency that should be fix in the Makefile instead of working around it here. I don't know what the fix will look like, but semi-randomly replicated `$` multiple times doesn't look like the right approach for this problem.